recipe/conda_build_config.yaml

```diff
@@ -1,3 +1,4 @@
+build_platform: {osx_arm64: osx_64, linux_ppc64le: linux_64, linux_aarch64: linux_64}
```

Comment:
This should be in `conda-forge.yml` file at the root of the repo

Reply:
Done in c2c051059be4defb2908cd846c8270a2299ac4b1.